.eslintrc

```diff
@@ -1,51 +1,51 @@
 {
   "root": true,
   "parser": "@typescript-eslint/parser",
-  "plugins": ["import", "prettier", "react-hooks", "@typescript-eslint"],
+  "plugins": ["import", "prettier", "react-hooks", "@typescript-eslint", "unused-imports"],
   "env": {
     "node": true
   },
   "extends": [
-    "prettier",
     "eslint:recommended",
     "plugin:@typescript-eslint/eslint-recommended",
-    "plugin:@typescript-eslint/recommended"
+    "plugin:@typescript-eslint/recommended",
+    "plugin:prettier/recommended"
   ],
   "rules": {
     "arrow-body-style": ["error", "as-needed"],
     "react/react-in-jsx-scope": "off",
-    "no-trailing-spaces": "error",
     "react/no-children-prop": "off",
     "no-console": ["warn", { "allow": ["warn", "error"] }],
-    "quotes": ["error", "single"],
     "import/order": [
       "error",
-        {
-          "groups": [
-            ["external", "builtin"],
-            "internal",
-            ["sibling", "parent"],
-            "index"
-          ],
-          "pathGroups": [
-            {
-              "pattern":
-                  "@(api|assets|modules|redux|routes|shared|resources)/**",
-              "group": "internal"
-            },
-            {
-              "pattern":
-                  "@(redux|routes|resources|api|i18n)",
-              "group": "internal"
-            }
-          ],
-          "pathGroupsExcludedImportTypes": ["internal"],
-          "newlines-between": "always"
+      {
+        "groups": [["external", "builtin"], "internal", ["sibling", "parent"], "index"],
+        "pathGroups": [
+          {
```

Comment:
I believe that the empty line between the React import and other imports from libraries, as well as the alphabetized ordering of imports, may be considered excessive. Would you be able to provide some insight into the reasoning behind these choices?

Reply:
It leads to changes almost all the files and leads to investigate more time to figure out who did the initial changes and why.

Reply:
I took these rules from the `mindlogger-app-refactor` repo.

https://github.com/ChildMindInstitute/mindlogger-app-refactor/blob/a5c8f472cfefe2c5db6ccd0d02efafe73381d8d8/.eslintrc.js#L42-L46

Part of this effort is to bring the code styles in line with each other, to lower the cognitive load of switching between repos. I don't have a preference either way because the tools will auto-format the files which means we won't have to think about it.

Reply:
Or maybe it was the `mindlogger-web-refactor`, I don't remember but they both use it

Reply:
As it turns out, alphabetized imports break the test files because they currently depend on import order, so I've reverted this for now

Reply:
Thank you, and could you please revert adding empty line after `react` imports?